lib/Pusher.php

```diff
@@ -330,6 +352,7 @@ public function trigger( $channels, $event, $data, $socket_id = null, $debug = f
 
 		if ( $socket_id !== null )
 		{
+			$this->validate_socket_id( $socket_id );
```

Comment:
What do you think about doing validation at the top of the function rather than here? It's generally a good idea to fail fast and people expect to see validation at the entry point.